modules/op-changing-default-service-account.adoc

```diff
@@ -0,0 +1,25 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-changing-default-service-account_{context}"]
+= Changing the default service account
+
+You can change the default service account by editing the `default-service-account` field in the `.spec.pipeline` and `.spec.trigger` specifications. The default service account name is `pipeline`.
```

Comment:
```suggestion
You can change the default service account for Tekton by editing the `default-service-account` field in the `.spec.pipeline` and `.spec.trigger` specifications. The default service account name is `pipeline`.
```

not sure if this is right, might want to check with your eng team, but I'd clarify here what the default service account is being changed for